integration-test/679-less-landuse-building-label-placements.py

```diff
@@ -0,0 +1,21 @@
+# The landuse for a pier
+# http://www.openstreetmap.org/way/82206919
+assert_no_matching_feature(
+    14, 2620, 6330, 'landuse',
+    {'id': 82206919, 'kind': 'pier', 'label_placement': None})
```

Comment:
To check that a property doesn't exist, you should check for `type(None)`. Otherwise I believe it will try to match that it exists with a value of null. For example:

`{'id': 82206919, 'kind': 'pier', 'label_placement': type(None)}`